packages/tailwind/src/theme.ts

```diff
@@ -8,8 +8,8 @@ const theme: ThemeOptions = {
     outlined: ""
   },
   sizes: {
-    xs: "h-2 w-2",
-    sm: "h-4 w-4",
+    xs: "h-2",
```

Comment:
I don't get why these two lost the w while the others kept it.

Reply:
Keeping both on small size was making icons disappearing on small screen. We should investigate that on future